src/main/java/Gene/command/EventCommand.java

```diff
@@ -0,0 +1,17 @@
+package Gene.command;
+
+import Gene.task.Event;
+import Gene.GeneException;
+import Gene.task.TaskList;
+
+public class EventCommand {
+    public static void execute(String command, TaskList taskList) throws GeneException {
+        String[] parts = command.replaceFirst("\\S+", "").split("/");
+        if (parts.length < 3) {
+            throw new GeneException("Invalid event format." + System.lineSeparator() + "Use Format: event (Task Description) /from (Date) /to (Date)");
```

Comment:
Try to make sure that the line length is smaller than 120 chars. Perhaps you can line wrap it.